tests/e2e/contracts/log_indexer_testnet.py

```
@@ -0,0 +1,67 @@
+class LogIndexer(gl.Contract):
+    vector_store: VecDB[np.float32, typing.Literal[384], StoreValue]
+
+    def __init__(self):
+        pass
+
+    def get_embedding_generator(self):
+        return genlayermodelwrappers.SentenceTransformer("all-MiniLM-L6-v2")
+
+    def get_embedding(
+        self, txt: str
+    ) -> np.ndarray[tuple[typing.Literal[384]], np.dtypes.Float32DType]:
```

Pylint (3.3.7)

[error] Value 'np.ndarray' is unsubscriptable

(E1136)